src/frontends/paddle/src/op/p_norm.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs p_norm(const NodeContext& node) {
+    auto data = node.get_input("X");
+    auto p = node.get_attribute<float>("porder");
+    auto axis = node.get_attribute<int32_t>("axis");
+    auto keepdim = node.get_attribute<bool>("keepdim");
```

Comment:
better use the default value for 'get_attribute()' function in case it throw error for models which doesn't set this attributes

Reply:
down